### PR TITLE
Merkle User ID Module: updates to user id submodule

### PR DESCRIPTION
**Pull request by SKOCHERI**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: modules/merkleIdSystem.js [hunk 1]
```diff
@@ -8,8 +8,40 @@
...
+
+function readFromLocalStorage() {
+  return storage.localStorageIsEnabled() ? storage.getDataFromLocalStorage(SESSION_COOKIE_NAME) : null;
+}
+
+function constructUrl(configParams) {
+  const session = getSession(configParams);
+  let url = `${configParams.endpoint}?vendor=${configParams.vendor}&sv_cid=${configParams.sv_cid}&sv_domain=${configParams.sv_domain}&sv_pubid=${configParams.sv_pubid}`;
+  if (session) {
+    url.append(`?sv_session=${session}`);
```
Comment (smenzer) [inline modules/merkleIdSystem.js:40]: i think this start with an `&` and not a `?` since it's not the first querystring param in the list

**Review by smenzer — CHANGES_REQUESTED** [review]
generally looks good, with one comment below.

also needs a docs PR since there are publisher-facing changes here. this is also a breaking change for publishers since there are new values that are required now...should this move to the 5.0 branch?

**Comment by angelamerkelprebid** [comment]
My reign is over

**Comment by jdwieland8282** [comment]
Spoke with @angelamerkelprebid in the 3/31 pbjs PMC.   This module was originally built by an external party.  This is not an official userid sub adapter, merkle does not intend for their module to work this way.  This PR represents the changes Merkle has requested be made.

**Comment by smenzer** [comment]
thanks @jdwieland8282 very helpful context.

**Review by smenzer — APPROVED** [review]
LGTM

just needs docs

**Review by patmmccann — APPROVED** [review]

**Comment by jdwieland8282** [comment]
docs pr https://github.com/prebid/prebid.github.io/pull/2822

**Pull request merged**